llvm/lib/Target/RISCV/GISel/RISCVInstructionSelector.cpp

```diff
@@ -379,6 +381,30 @@ RISCVInstructionSelector::selectSHXADD_UWOp(MachineOperand &Root,
   return std::nullopt;
 }
 
+InstructionSelector::ComplexRendererFns
+RISCVInstructionSelector::renderVLOp(MachineOperand &Root) const {
+  MachineRegisterInfo &MRI =
```

Comment:
This should be a member of RISCVInstructionSelector if it's not already

Reply:
It's not. Is it okay for me to do this as a follow up? There are many other functions in this file that are getting MRI like this that need refactoring too.

Reply:
Yes